web/src/stream_edit.ts

```diff
@@ -167,49 +210,53 @@ function show_subscription_settings(sub) {
     });
 }
 
-function has_global_notification_setting(setting_label) {
-    if (setting_label.includes("_notifications")) {
-        return true;
-    } else if (setting_label.includes("_notify")) {
-        return true;
-    }
-    return false;
-}
-
-function is_notification_setting(setting_label) {
-    return has_global_notification_setting(setting_label) || setting_label === "is_muted";
+function is_notification_setting(setting_label: string): boolean {
+    return (
+        notification_labels_schema.safeParse(setting_label).success || setting_label === "is_muted"
+    );
 }
 
-export function stream_settings(sub) {
+export function stream_settings(sub: StreamSubscription): StreamSetting[] {
```

Comment:
This function is a bit messy with all these zod parsing things, but maybe that's what's needed? Curious what Anders thinks